docs/src/examples/AccordionExample/AccordionExampleFirst.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Accordion, AccordionItemProps, DisplayText, Label } from '../../../../src/components/';
+import * as styles from '../../styles/components-page.scss';
+
+export interface Props {};
+
+export interface State {
+  Item1: boolean,
+  Item2: boolean,
+  Item3: boolean
+};
+
+class AccordionExample extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      Item1: false,
+      Item2: false,
+      Item3: false
+    };
+
```

Comment:
I feel that we can take items as an array, instead of defining each items separately. If there are 10 items, we don't want to define 10 items in state.